parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -224,16 +223,15 @@ pretty0
           <> optSpace <> (fmt S.DelimiterChar $ l "]")
       where optSpace = PP.orElse "" " "
     If' cond t f -> paren (p >= 2) $
-      if height > 0 then PP.lines [
+      if PP.isMultiLine pt || PP.isMultiLine pf then PP.lines [
         (fmt S.ControlKeyword "if ") <> pcond <> (fmt S.ControlKeyword " then") `PP.hang` pt,
         (fmt S.ControlKeyword "else") `PP.hang` pf
        ]
       else PP.spaced [
-        (fmt S.ControlKeyword "if") `PP.hang` pcond <> ((fmt S.ControlKeyword " then") `PP.hang` pt),
+        ((fmt S.ControlKeyword "if") `PP.hang` pcond) <> ((fmt S.ControlKeyword " then") `PP.hang` pt),
```

Comment:
Oops, I left this in while investigating the relationship between infix `PP.hang` and `<>`. Should I remove it?

Reply:
I'd keep it.